debian/clearwater-socket-factory.postinst

```diff
@@ -0,0 +1,162 @@
+#!/bin/sh
```

Comment:
I think you need to write this for CentOS too.  We tend to pull this into a script in `/usr/share/clearwater/infrastructure/install` and then run it from here and from the `clearwater-socket-factory.spec` file.